torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14965,22 +15030,26 @@ def ref_pairwise_distance(input1, input2):
         nan_policy='propagate',
         dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
         ref=reference_reduction_numpy(np.amax),
+        sample_inputs_func=sample_inputs_amax_amin_reduction,
         skips=(
             # FIXME: sum reduces all dimensions when dim=[]
             DecorateInfo(unittest.expectedFailure, 'TestReductions', 'test_dim_empty'),
             DecorateInfo(unittest.expectedFailure, 'TestReductions', 'test_dim_empty_keepdim'),
         ),
+        error_inputs_func=error_inputs_aminmax_amax_amin,
     ),
     ReductionOpInfo(
         'amin',
         nan_policy='propagate',
         dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
         ref=reference_reduction_numpy(np.amin),
+        sample_inputs_func=sample_inputs_amax_amin_reduction,
```

Comment:
ReductionOpInfos come with sample inputs function that generates a number of inputs, please don't overwrite this function.

Reply:
I think the inputs for these ReductionOpInfos are generated from: 
(Please let me know if I'm wrong)

https://github.com/pytorch/pytorch/blob/1e64c8a8e37aceb82675b77afd71fd8fc78cd0bb/torch/testing/_internal/common_methods_invocations.py#L949-L954

I see the cases involving zero dimensions are missing
```python
SampleInput(make_tensor((S, 0, S), dtype=dtype, device=device, requires_grad=requires_grad), kwargs={'dim': -1})
```

I thought making amendments to this might break other ops too. So I added another arg.

But I removed it if this way is not correct. Any other way to that input? Thanks!